### PR TITLE
Update uTorrent.xml

### LINKED ISSUE #5830: Update uTorrent
I don't see any reason to block ``www`` (cert seems legit).

### PR DESCRIPTION
**Pull request by luixxiul**
- Edit www subdomain (cert seems legit)
- Added other subdomains

Closes #5830

### CODE REVIEW

**Comment by J0WI** [comment]
You forgot utorrent.com (without www) :)
There are just some trivial mixed content issues, so rule lgtm.

**Comment by J0WI** [comment]
btw: you don't need to open an issue for every PR.

**Comment by luixxiul** [comment]
@J0WI OK I see

**Comment by luixxiul** [comment]
Done.

File: src/chrome/content/rules/uTorrent.xml [hunk 1]
```diff
@@ -16,22 +15,30 @@
...
-
-	<rule from="^http://(forum|remote)\.utorrent\.com/"
-		to="https://$1.utorrent.com/" />
-
-	<rule from="^http://(?:ll\.)?(download3?|static)\.utorrent\.com/"
-		to="https://$1.utorrent.com/" />
+<ruleset name="µTorrent">
+
+	<target host="utorrent.com" />
+    <target host="www.utorrent.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/uTorrent.xml]: code style (use tabs)

Comment (luixxiul) [inline src/chrome/content/rules/uTorrent.xml]: that's why: https://github.com/EFForg/https-everywhere/issues/5837 :-)

**Comment by luixxiul** [comment]
Updated.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**